docs/program/sdks/_index.md

```diff
@@ -1,355 +1,15 @@
 ---
-title: "Program your Robot with Viam's SDKs"
-linkTitle: "Viam SDKs"
-weight: 40
+title: "Interact with Resources with Viam's SDKs"
+linkTitle: "SDK API Libraries"
```

Comment:
This and its sub-pages were probably a good amount of work to put together, but honestly I don't think we need/want the sub-pages.  It would seem to be just more more place we'd need to maintain the APIs, and I think its actually less user-friendly in one big page vs where it already is embedded in each resource page...

Reply:
Might take me a bit to test out but that's a valid suggestion! I like splitting the Robot page out

Reply:
Ok done @mcvella @npentrel